cpp/models/abm/person.h

```diff
@@ -68,6 +69,15 @@ class Person
         return (m_person_id == other.m_person_id);
     }
 
+    /**
+     * @brief The TestResult of a Person.
+     */
+    struct TestResult {
+        TimePoint time_of_testing; ///< The TimePoint when the Person performs the test.
+        TestType type = TestType::Count; ///< The TestType of the test.
+        bool is_allowed_to_enter; ///< Whether the person is allowed to enter the Location.
```

Comment:
I would probably rather save the pure test result here, i.e., positive or negative (a renaming should thus suffice).

Reply:
Yes, I updated it and the `run_scheme()` function.